### ISSUE TITLE
Missing C template

### ISSUE THREAD

**Issue by troycurtisjr** [LucHermitte/mu-template#1]
`c/internals/register-file-kinds.template` references `c/internals/register-file-kinds-spe`, but that file does not exist.

**Comment by LucHermitte** [comment]
Yes this is a feature.

This a variation point -- that I wanted to follow the _Strategy_ design pattern. The idea is that we can use it to inject new sub-filetypes on a per-project basis.

e.g. in a new project (defined with local_vimrc for instance), you can have new template-files and among them one can be used to tell which pattern unit-test files follow.

Unless I've messed up mu-template internals, you should not observe any error message.

**Comment by troycurtisjr** [comment]
![mutemplatescreenshot 2015-12-28 22 00 13](https://cloud.githubusercontent.com/assets/16273537/12029350/78c31af2-adae-11e5-803c-027c5bc4d027.png) This is the error that I saw that prompted the issue report.  It didn't really stop anything from happening this time, but seeing the error message go by made me wonder what the trouble was.  Also it does seem like the last time it paused and I had to hit `Enter` to continue, but this time it went by just leaving the `dir=....` message in the status area.
